tests/test_globus.py

```diff
@@ -0,0 +1,184 @@
+import configparser
+import os
+import re
+import shutil
+import socket
+import unittest
+
+from fair_research_login.client import NativeClient
+from globus_sdk import DeleteData, TransferClient
+from globus_sdk.exc import TransferAPIError
+
+from tests.base import TOP_LEVEL, ZSTASH_PATH, TestZstash, print_starred, run_cmd
+
+# Use 'Globus Tutorial Endpoint 1' to simulate an HPSS Globus endpoint
+hpss_globus_endpoint = "ddb59aef-6d04-11e5-ba46-22000b92c6ec"
```

Comment:
When I run this test I get `AssertionError: The ddb59aef-6d04-11e5-ba46-22000b92c6ec endpoint is not activated. Please go to https://app.globus.org/file-manager/collections/ddb59aef-6d04-11e5-ba46-22000b92c6ec and activate the endpoint.` I don't see a "activate endpoint" button for this endpoint.

Reply:
Thank you for catching it. I have updated zstash/globus.py and tests/test_globus.py to fix it, https://github.com/E3SM-Project/zstash/pull/154/commits/56d1f451a0dd09ac4aeab2e9982935a77321d6ad